src/Psalm/Codebase.php

```diff
@@ -1279,10 +1279,13 @@ public function getCompletionDataAtPosition(string $file_path, Position $positio
         }
 
         foreach ($reference_map as $start_pos => [$end_pos, $possible_reference]) {
-            if ($offset < $start_pos || $possible_reference[0] !== '*') {
```

Comment:
As mentioned below, I'm not sure why this is only allowing refs that are tagged with `*` at the start!